File: clients/csharp/examples/tables-list-metadata-default.cs
using System.Text.Json;
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get a database
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      "**API_ENDPOINT**",
      "**APPLICATION_TOKEN**"
    );

    // List table metadata
    var result = await database.ListTablesAsync();

    Console.WriteLine(JsonSerializer.Serialize(result));
  }
}

File: clients/java/examples/tables-list-metadata-default.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import com.datastax.astra.client.tables.definition.TableDescriptor;
import java.util.List;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // List table metadata
    List<TableDescriptor> result = database.listTables();

    System.out.println(result);
  }
}
